packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjects.js

```diff
@@ -1,30 +1,20 @@
 import { Anchor, Box, ResponsiveContext, Text } from 'grommet'
 import { arrayOf, bool, shape, string } from 'prop-types'
 import { useContext } from 'react'
-import { Loader, SpacedHeading, SpacedText } from '@zooniverse/react-components'
+import { Loader, SpacedText } from '@zooniverse/react-components'
 
+import { ContentBox } from '@components/shared'
 import SubjectCard from '../SubjectCard/SubjectCard.js'
 
 function RecentSubjects({
   isLoading = false,
   recents = [],
-  recentsError = undefined
+  error = undefined
 }) {
   const size = useContext(ResponsiveContext)
 
   return (
-    <Box
-      pad='medium'
-      height={{ min: '200px' }}
-      round='small'
-      border={{
-        color: { light: 'light-5', dark: 'black' },
-        size: 'xsmall'
-      }}
-    >
-      <SpacedHeading size='1.125rem' level={2} margin={{ top: '0' }}>
-        Recent Classifications
-      </SpacedHeading>
+    <ContentBox title='Recent Classifications' screenSize={size}>
```

Comment:
When I first built RecentSubjects, I didn't realize ContentBox was available for use, so I changed it here!